docs/source/advanced/test-builders.mdx

```diff
@@ -1,38 +1,32 @@
 ---
-title: 'Test Builders'
+title: Test builders
 ---
 
-> Note: Test Builders are experimental and subject to change. We'd love to have your feedback on them. Things that worked well, things that can be improved, etc.. Feel free to reach out either through [Github issues](https://github.com/apollographql/apollo-android/issues/new?assignees=&labels=Type%3A+Bug&template=bug_report.md&title=[Test Builders]) or the [Kotlinlang slack channel](https://app.slack.com/client/T09229ZC6/C01A6KM1SBZ).
+> ⚠️ **Test builders are experimental and subject to change.** If you have feedback on them, please let us know via [GitHub issues](https://github.com/apollographql/apollo-android/issues/new?assignees=&labels=Type%3A+Bug&template=bug_report.md&title=[Test%20Builders]) or the [Apollo forums](https://community.apollographql.com/).
 
-Apollo Android generates model classes based on your operations. You can then use these classes in your
-project, which means you will probably need to instantiate them in your tests.
+Apollo Android provides **test builders** that enable you to instantiate generated GraphQL model classes with default values. Test builders are especially helpful for testing models with a large number of fields or a deeply nested hierarchy. Additionally, they automatically populate the `__typename` field and deduplicate merged fields whenever possible.
 
-However, if your models have many fields or their hierarchy is deep, you may find it tedious to instantiate them
-manually - as a value must be set for each field.
+## Enabling test builders
 
-To make it easier, Apollo Android provides Test Builders: an optional mechanism that helps to build these models, with default values.
-Additionally, Test Builders will populate `__typename` for you and deduplicate merged fields when possible.
-
-As it may not be useful to all users and it generates more code, test builder generation is not enabled by default. To
-enable it, set the `generateTestBuilders` option to `true`:
+Test builders are _not_ enabled by default, because they aren't useful for every application and they generate additional code. To enable them, set the `generateTestBuilders` option to `true`:
 
 ```kotlin:title=build.gradle[.kts]
 apollo {
   // ...
 
   // Enable test builder generation
   generateTestBuilders.set(true)
-  
-  // Optionally, you may customize the output path
+
+  // Optionally, you can customize the output path
   testDir.set(file("test-builders"))
```

Comment:
I think we can remove that option as it's not needed for most use cases?

Reply:
Totally, no objection to remove it.